Deployment fails with DB2

This error is thrown when trying to deploy `ticktock` stream from a dataflow. As other db's are working fine, this has to be something db2 related:
```
2019-02-11 10:14:30.658  INFO 8150 --- [eTaskExecutor-3] o.s.c.d.spi.local.LocalAppDeployer       : Deploying app with deploymentId ticktock.log-v1 instance 0.
   Logs will be in /tmp/spring-cloud-deployer-7676249310598120960/ticktock-1549880070554/ticktock.log-v1
2019-02-11 10:14:30.759  WARN 8150 --- [eTaskExecutor-3] o.h.engine.jdbc.spi.SqlExceptionHelper   : SQL Error: -302, SQLState: 22001
2019-02-11 10:14:30.760 ERROR 8150 --- [eTaskExecutor-3] o.h.engine.jdbc.spi.SqlExceptionHelper   : DB2 SQL Error: SQLCODE=-302, SQLSTATE=22001, SQLERRMC=null, DRIVER=4.19.26
2019-02-11 10:14:30.763 ERROR 8150 --- [eTaskExecutor-3] o.h.i.ExceptionMapperStandardImpl        : HHH000346: Error during managed flush [org.hibernate.exception.DataException: could not execute statement]
2019-02-11 10:14:30.776  INFO 8150 --- [eTaskExecutor-3] o.s.s.support.AbstractStateMachine       : Interceptors threw exception, skipping state change

org.springframework.statemachine.StateMachineException: Unable to persist stateMachineContext; nested exception is org.springframework.dao.DataIntegrityViolationException: could not execute statement; SQL [n/a]; nested exception is org.hibernate.exception.DataException: could not execute statement
	at org.springframework.statemachine.persist.AbstractPersistingStateMachineInterceptor.preStateChange(AbstractPersistingStateMachineInterceptor.java:80) ~[spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.config.AbstractStateMachineFactory$RegionPersistingInterceptorAdapter.preStateChange(AbstractStateMachineFactory.java:408) ~[spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.StateMachineInterceptorList.preStateChange(StateMachineInterceptorList.java:108) ~[spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.AbstractStateMachine.callPreStateChangeInterceptors(AbstractStateMachine.java:880) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.AbstractStateMachine.switchToState(AbstractStateMachine.java:911) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.AbstractStateMachine.access$500(AbstractStateMachine.java:82) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.AbstractStateMachine$3.transit(AbstractStateMachine.java:337) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.DefaultStateMachineExecutor.handleTriggerTrans(DefaultStateMachineExecutor.java:288) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.DefaultStateMachineExecutor.executeTriggerlessTransitions(DefaultStateMachineExecutor.java:475) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.AbstractStateMachine.executeTriggerlessTransitions(AbstractStateMachine.java:825) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.AbstractStateMachine$2.onComplete(AbstractStateMachine.java:265) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.state.CompositeStateListener.onComplete(CompositeStateListener.java:51) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.state.AbstractState.notifyStateOnComplete(AbstractState.java:535) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.state.AbstractState.scheduleStateActions(AbstractState.java:484) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.state.AbstractState.entry(AbstractState.java:269) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.state.ObjectState.entry(ObjectState.java:163) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.AbstractStateMachine.entryToState(AbstractStateMachine.java:1310) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.AbstractStateMachine.entryToState(AbstractStateMachine.java:1254) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.AbstractStateMachine.setCurrentStateInternal(AbstractStateMachine.java:1060) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.AbstractStateMachine.setCurrentState(AbstractStateMachine.java:1034) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.AbstractStateMachine.switchToState(AbstractStateMachine.java:926) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.AbstractStateMachine.access$500(AbstractStateMachine.java:82) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.AbstractStateMachine$3.transit(AbstractStateMachine.java:334) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.DefaultStateMachineExecutor.handleInitialTrans(DefaultStateMachineExecutor.java:303) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.DefaultStateMachineExecutor.processTriggerQueue(DefaultStateMachineExecutor.java:399) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.DefaultStateMachineExecutor.access$200(DefaultStateMachineExecutor.java:65) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.DefaultStateMachineExecutor$1.run(DefaultStateMachineExecutor.java:331) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.core.task.SyncTaskExecutor.execute(SyncTaskExecutor.java:50) [spring-core-5.1.3.RELEASE.jar!/:5.1.3.RELEASE]
	at org.springframework.statemachine.support.DefaultStateMachineExecutor.scheduleEventQueueProcessing(DefaultStateMachineExecutor.java:354) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.DefaultStateMachineExecutor.execute(DefaultStateMachineExecutor.java:163) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.DefaultStateMachineExecutor.doStart(DefaultStateMachineExecutor.java:175) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.LifecycleObjectSupport.start(LifecycleObjectSupport.java:120) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.AbstractStateMachine.doStart(AbstractStateMachine.java:418) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.LifecycleObjectSupport.start(LifecycleObjectSupport.java:120) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.state.StateMachineState.entry(StateMachineState.java:236) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.AbstractStateMachine.entryToState(AbstractStateMachine.java:1310) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.AbstractStateMachine.entryToState(AbstractStateMachine.java:1254) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.AbstractStateMachine.setCurrentStateInternal(AbstractStateMachine.java:1060) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.AbstractStateMachine.setCurrentState(AbstractStateMachine.java:1034) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.AbstractStateMachine.switchToState(AbstractStateMachine.java:926) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.AbstractStateMachine.access$500(AbstractStateMachine.java:82) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.AbstractStateMachine$3.transit(AbstractStateMachine.java:337) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.DefaultStateMachineExecutor.handleTriggerTrans(DefaultStateMachineExecutor.java:288) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.DefaultStateMachineExecutor.handleTriggerTrans(DefaultStateMachineExecutor.java:211) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.DefaultStateMachineExecutor.processTriggerQueue(DefaultStateMachineExecutor.java:451) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.DefaultStateMachineExecutor.access$200(DefaultStateMachineExecutor.java:65) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.support.DefaultStateMachineExecutor$1.run(DefaultStateMachineExecutor.java:325) [spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [na:1.8.0_181]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [na:1.8.0_181]
	at java.lang.Thread.run(Thread.java:748) [na:1.8.0_181]
Caused by: org.springframework.dao.DataIntegrityViolationException: could not execute statement; SQL [n/a]; nested exception is org.hibernate.exception.DataException: could not execute statement
	at org.springframework.orm.jpa.vendor.HibernateJpaDialect.convertHibernateAccessException(HibernateJpaDialect.java:300) ~[spring-orm-5.1.3.RELEASE.jar!/:5.1.3.RELEASE]
	at org.springframework.orm.jpa.vendor.HibernateJpaDialect.translateExceptionIfPossible(HibernateJpaDialect.java:253) ~[spring-orm-5.1.3.RELEASE.jar!/:5.1.3.RELEASE]
	at org.springframework.orm.jpa.JpaTransactionManager.doCommit(JpaTransactionManager.java:536) ~[spring-orm-5.1.3.RELEASE.jar!/:5.1.3.RELEASE]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.processCommit(AbstractPlatformTransactionManager.java:746) ~[spring-tx-5.1.3.RELEASE.jar!/:5.1.3.RELEASE]
	at org.springframework.transaction.support.AbstractPlatformTransactionManager.commit(AbstractPlatformTransactionManager.java:714) ~[spring-tx-5.1.3.RELEASE.jar!/:5.1.3.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.commitTransactionAfterReturning(TransactionAspectSupport.java:533) ~[spring-tx-5.1.3.RELEASE.jar!/:5.1.3.RELEASE]
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:304) ~[spring-tx-5.1.3.RELEASE.jar!/:5.1.3.RELEASE]
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:98) ~[spring-tx-5.1.3.RELEASE.jar!/:5.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.1.3.RELEASE.jar!/:5.1.3.RELEASE]
	at org.springframework.dao.support.PersistenceExceptionTranslationInterceptor.invoke(PersistenceExceptionTranslationInterceptor.java:139) ~[spring-tx-5.1.3.RELEASE.jar!/:5.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.1.3.RELEASE.jar!/:5.1.3.RELEASE]
	at org.springframework.data.jpa.repository.support.CrudMethodMetadataPostProcessor$CrudMethodMetadataPopulatingMethodInterceptor.invoke(CrudMethodMetadataPostProcessor.java:135) ~[spring-data-jpa-2.1.3.RELEASE.jar!/:2.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.1.3.RELEASE.jar!/:5.1.3.RELEASE]
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:93) ~[spring-aop-5.1.3.RELEASE.jar!/:5.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.1.3.RELEASE.jar!/:5.1.3.RELEASE]
	at org.springframework.data.repository.core.support.SurroundingTransactionDetectorMethodInterceptor.invoke(SurroundingTransactionDetectorMethodInterceptor.java:61) ~[spring-data-commons-2.1.3.RELEASE.jar!/:2.1.3.RELEASE]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186) ~[spring-aop-5.1.3.RELEASE.jar!/:5.1.3.RELEASE]
	at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:212) ~[spring-aop-5.1.3.RELEASE.jar!/:5.1.3.RELEASE]
	at com.sun.proxy.$Proxy172.save(Unknown Source) ~[na:na]
	at org.springframework.statemachine.data.RepositoryStateMachinePersist.write(RepositoryStateMachinePersist.java:67) ~[spring-statemachine-data-common-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.data.jpa.JpaPersistingStateMachineInterceptor.write(JpaPersistingStateMachineInterceptor.java:65) ~[spring-statemachine-data-jpa-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	at org.springframework.statemachine.persist.AbstractPersistingStateMachineInterceptor.preStateChange(AbstractPersistingStateMachineInterceptor.java:78) ~[spring-statemachine-core-2.1.0.BUILD-SNAPSHOT.jar!/:2.1.0.BUILD-SNAPSHOT]
	... 49 common frames omitted
Caused by: org.hibernate.exception.DataException: could not execute statement
	at org.hibernate.exception.internal.SQLExceptionTypeDelegate.convert(SQLExceptionTypeDelegate.java:52) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	at org.hibernate.exception.internal.StandardSQLExceptionConverter.convert(StandardSQLExceptionConverter.java:42) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:113) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	at org.hibernate.engine.jdbc.spi.SqlExceptionHelper.convert(SqlExceptionHelper.java:99) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:178) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	at org.hibernate.persister.entity.AbstractEntityPersister.update(AbstractEntityPersister.java:3356) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	at org.hibernate.persister.entity.AbstractEntityPersister.updateOrInsert(AbstractEntityPersister.java:3229) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	at org.hibernate.persister.entity.AbstractEntityPersister.update(AbstractEntityPersister.java:3630) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	at org.hibernate.action.internal.EntityUpdateAction.execute(EntityUpdateAction.java:146) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:604) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:478) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:356) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:39) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	at org.hibernate.internal.SessionImpl.doFlush(SessionImpl.java:1454) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	at org.hibernate.internal.SessionImpl.managedFlush(SessionImpl.java:511) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	at org.hibernate.internal.SessionImpl.flushBeforeTransactionCompletion(SessionImpl.java:3283) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	at org.hibernate.internal.SessionImpl.beforeTransactionCompletion(SessionImpl.java:2479) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	at org.hibernate.engine.jdbc.internal.JdbcCoordinatorImpl.beforeTransactionCompletion(JdbcCoordinatorImpl.java:473) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl.beforeCompletionCallback(JdbcResourceLocalTransactionCoordinatorImpl.java:178) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl.access$300(JdbcResourceLocalTransactionCoordinatorImpl.java:39) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	at org.hibernate.resource.transaction.backend.jdbc.internal.JdbcResourceLocalTransactionCoordinatorImpl$TransactionDriverControlImpl.commit(JdbcResourceLocalTransactionCoordinatorImpl.java:271) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	at org.hibernate.engine.transaction.internal.TransactionImpl.commit(TransactionImpl.java:98) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	at org.springframework.orm.jpa.JpaTransactionManager.doCommit(JpaTransactionManager.java:532) ~[spring-orm-5.1.3.RELEASE.jar!/:5.1.3.RELEASE]
	... 68 common frames omitted
Caused by: com.ibm.db2.jcc.am.SqlDataException: DB2 SQL Error: SQLCODE=-302, SQLSTATE=22001, SQLERRMC=null, DRIVER=4.19.26
	at com.ibm.db2.jcc.am.kd.a(Unknown Source) ~[db2jcc4-4.19.26.jar!/:na]
	at com.ibm.db2.jcc.am.kd.a(Unknown Source) ~[db2jcc4-4.19.26.jar!/:na]
	at com.ibm.db2.jcc.am.kd.a(Unknown Source) ~[db2jcc4-4.19.26.jar!/:na]
	at com.ibm.db2.jcc.am.fp.b(Unknown Source) ~[db2jcc4-4.19.26.jar!/:na]
	at com.ibm.db2.jcc.am.fp.c(Unknown Source) ~[db2jcc4-4.19.26.jar!/:na]
	at com.ibm.db2.jcc.t4.bb.l(Unknown Source) ~[db2jcc4-4.19.26.jar!/:na]
	at com.ibm.db2.jcc.t4.bb.a(Unknown Source) ~[db2jcc4-4.19.26.jar!/:na]
	at com.ibm.db2.jcc.t4.p.a(Unknown Source) ~[db2jcc4-4.19.26.jar!/:na]
	at com.ibm.db2.jcc.t4.wb.b(Unknown Source) ~[db2jcc4-4.19.26.jar!/:na]
	at com.ibm.db2.jcc.am.gp.tc(Unknown Source) ~[db2jcc4-4.19.26.jar!/:na]
	at com.ibm.db2.jcc.am.gp.b(Unknown Source) ~[db2jcc4-4.19.26.jar!/:na]
	at com.ibm.db2.jcc.am.gp.b(Unknown Source) ~[db2jcc4-4.19.26.jar!/:na]
	at com.ibm.db2.jcc.am.gp.lc(Unknown Source) ~[db2jcc4-4.19.26.jar!/:na]
	at com.ibm.db2.jcc.am.gp.executeUpdate(Unknown Source) ~[db2jcc4-4.19.26.jar!/:na]
	at com.zaxxer.hikari.pool.ProxyPreparedStatement.executeUpdate(ProxyPreparedStatement.java:61) ~[HikariCP-3.2.0.jar!/:na]
	at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeUpdate(HikariProxyPreparedStatement.java) ~[HikariCP-3.2.0.jar!/:na]
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:175) ~[hibernate-core-5.3.7.Final.jar!/:5.3.7.Final]
	... 86 common frames omitted

2
```

--- a/spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/db/migration/db2/V1__Initial_Setup.java
+++ b/spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/db/migration/db2/V1__Initial_Setup.java
@@ -182,7 +182,7 @@ public class V1__Initial_Setup extends AbstractInitialSetupMigration {
 			"create table state_machine (\n" +
 			"    machine_id varchar(255) not null,\n" +
 			"    state varchar(255),\n" +
-			"    state_machine_context blob(255),\n" +
+			"    state_machine_context blob(1048576),\n" +
 			"    primary key (machine_id)\n" +
 			")";
 

